cypress/integration/doiForm.spec.ts

```diff
@@ -1,15 +1,20 @@
+import { ObjectTypes } from "constants/wizardObject"
+
 describe("DOI form", function () {
   beforeEach(() => {
     cy.task("resetDb")
     cy.login()
-    cy.get("button", { timeout: 10000 }).contains("Create submission").click()
-
-    // Add folder name & description, navigate to submissions
-    cy.newSubmission()
   })
   it("should render DOI form correctly with formats' prefilled values and affiliation's autocomplete field", () => {
+    cy.generateFolderAndObjects(ObjectTypes.experiment)
+    cy.generateObject(ObjectTypes.dataset) // Needed for navigating to Publish -step
```

Comment:
Is it for testing or why don't we have `cy.generateFolderAndObjects()` to have all data readay as usual ?

Reply:
Good call. In this case I think it's sufficient to generate all objects.
Initial thought was to keep submitted objects count to bare minimum.

Reply:
This is now implemented.